http-native/build.gradle

```diff
@@ -129,3 +129,33 @@ compileJava {
         classpath = files()
     }
 }
+
+task extractJavaClassFiles(type: Copy) {
+    def tomlVersion = project.version.replace("-SNAPSHOT", "")
+    def jar = file("../http-ballerina-tests/build/target/extracted-distributions/jballerina-tools-zip/jballerina-tools-${project.ballerinaLangVersion}/repo/bala/ballerina/http/${tomlVersion}/platform/java11/http-native-${project.version}.jar")
```

Comment:
Can't we use the path from `build/lib` directory of http-native module

Reply:
yes, we can use. I will change the file path.